dojo/tools/api_sonarqube/importer.py

```diff
@@ -243,7 +251,16 @@ def import_hotspots(self, test):
                     continue
 
                 issue_type = "SECURITY_HOTSPOT"
-                severity = "Info"
+                if hotspot["vulnerabilityProbability"] == "CRITICAL":
+                    severity = "Medium"
```

Comment:
I suppose, this should not be `"Medium"` :)

Reply:
fixed it - thanks for checking. Sorry for that...